docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -14,8 +14,8 @@ UniquePersonList -right-> Person
 
 Person -up-> "*" Tag
 
-Person *--> Name
 Person *--> Email
+Person *--> Name
```

Comment:
I think Name should be before Email

Reply:
Oh i was just changing the lines for the UML diagram tracker thing right my bad

Reply:
This is cheating but... I guess we will ignore it.